debian/copyright

```diff
@@ -0,0 +1,38 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: coding-shell-extensions
+Source: <url://example.com>
+
+Files: *
+Copyright: <years> <put author's name and email here>
```

Comment:
So, we probably have to put licenses and copyrights for each extension. Care to check what the gnome-shell-extensions debian package does here?